app/src/org/commcare/activities/InstallFromListActivity.java

```diff
@@ -341,21 +333,21 @@ public void handleIOException(IOException exception) {
     }
 
     private void handleRequestError(int responseCode) {
-        System.out.println(responseCode);
+        System.out.println("Request to " + urlCurrentlyRequestingFrom + " had error code response: " + responseCode);
```

Comment:
Yea, I wasn't sure.. I've been trying lately to lean on the side of not cluttering the logs with things we probably won't ever act upon, but I may have gone too far in the other direction. This should probably be a log.